src/components/SponsorTimeEditComponent.tsx

```diff
@@ -320,6 +320,7 @@ class SponsorTimeEditComponent extends React.Component<SponsorTimeEditProps, Spo
     }
 
     changeTimesWhenScrolling(index: number, e: React.WheelEvent, sponsorTime: SponsorTime): void {
+        if (!Config.config.allowScrollingToEdit) return;
```

Comment:
are you fine with ignoring the event or would you rather not register `onWheel`?